index_bidding_history1.php

```diff
@@ -0,0 +1,134 @@
+  //       }
+  ?>  
+        </a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="/onlineauction/index.php">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Previous Page</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/onlineauction/logout.php">logout</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+
+    <div class="container-fuild">
+    
+        <table class="table">
+            <thead class="table-dark">
+              <tr><h1 style="text-align:center">Bidding history lists</h1></tr>
+                <tr>
+                    <th style="width:5%"> Bid ID</th>
+                    <th style="width:10%"> Product Name</th>
+                    <th> Product ID</th>
+                    <th style="width:15%"> Picture</th>
+                    <th> Bidding Status</th>
+                    <th> Your price</th>
+                    <th> Current Highest price</th>
+                    <th> Bidding Result</th>
+                    <th> Bidding End Date</th>
+                </tr>
+            </thead>
+            <tbody>
+              <?php
+                 $mail = $_SESSION['email'] ;
+                 $sql = "SELECT * FROM bidding JOIN item ON bidding.item_ID = item.item_ID JOIN users ON users.user_ID = bidding.buyer_ID WHERE users.user_type IN ('buyer', 'both') AND users.email = '$mail' order by item.item_id, bidding.bid_id";  # select data from sql table
+                 
+                 //$sql_maxPrice = "SELECT Max(bid_price) AS max_price, item_ID FROM bidding GROUP BY item_ID";
+
+
+                 // echo $sql;
+                 $result = mysqli_query($db_conn, $sql); # send a query to the database
+                 //$result_maxPrice = mysqli_query($db_conn, $sql_maxPrice);
+                 
+
+                 $resultCheck = mysqli_num_rows($result); # check if you can get the data from the databas
+                 //$resultCheck_maxPrice = mysqli_num_rows($result_maxPrice); 
+                 
+                // #fetch the data into an array
+                if ($resultCheck > 0) 
+                {
+                    while ($row = mysqli_fetch_assoc($result)) {
+
+                     $max_price = 0;
+                     $sql_maxPrice = "SELECT item.item_ID, max(bidding.bid_price) AS max_price FROM bidding JOIN item ON bidding.item_ID = item.item_ID JOIN users ON users.user_ID = bidding.buyer_ID WHERE users.user_type IN ('buyer', 'both') AND users.email != '$mail' group by item.item_ID order by item.item_id";  # select data from sql table 
```

Comment:
@yly828 As I have checked, you actually don't need this query and this while loop at all. 

You can move things out of this loop and use ` $max_price = maxBidQuery($row['item_ID'],$row['starting_price']);`